Kickstarter-iOS/Views/Cells/ThanksCategoryCell.swift

```diff
@@ -10,10 +10,10 @@ internal final class ThanksCategoryCell: UICollectionViewCell, ValueCell {
   @IBOutlet private weak var liveProjectCountLabel: UILabel!
 
   func configureWith(value category: KsApi.Category) {
-    self.bgView |> UIView.lens.backgroundColor .~ UIColorFromCategoryId(category.id) ?? .ksr_text_navy_900
+    self.bgView |> UIView.lens.backgroundColor .~ discoveryPrimaryColor(forCategoryId: category.root?.id)
```

Comment:
We were still using the old category colors here so I've updated this to use our current palette.